backend/submodule/rs_message.py

```diff
@@ -44,14 +44,17 @@ class RsMessageManager:
         RsMessage(309, RsMessageType.ERRO, "One or more banks need to enable ODT since ODT was a selected standard"),
         RsMessage(310, RsMessageType.ERRO, "Cascade requires prior bank termination"),
         RsMessage(311, RsMessageType.ERRO, "Load io error: {message}"),
+        # 404 Error Message
```

Comment:
What is this used for?

Reply:
restored the original rs_message.py file, the idea was to check the error which is not listed in rs_message.py, so I gave a thought of adding the name of the error as "Invalid error code" and give it 404. but for now I have written the tests for the errors that we have in rs_message.py